pulpcore/pulpcore/app/viewsets/base.py

```diff
@@ -33,11 +60,14 @@ class GenericNamedModelViewSet(viewsets.GenericViewSet):
             to django model filter expressions that can be used with the corresponding value from
             self.kwargs, used only by a nested ViewSet to filter based on the parent object's
             identity.
+        schema (rest_framework.schemas.AutoSchema): The schema class to use by default in
```

Comment:
qq: shouldn't this read `DefaultSchema`?

Reply:
By default it's a `DefaultSchema` but you can set this `schema` attribute to any subclass of `AutoSchema` in your subclass of `GenericNamedModelViewSet`.

Reply:
I'd say that it should document what it is rather than what it might be replaced with but this is a minor thing.

Reply:
@dparalen , fixed